README.md

```diff
@@ -335,3 +335,31 @@ val KernelBinary = cc.Binary{
 	Toolchain: kernelToolchain,
 }
 ```
+
+
+## Set up a local mirror
+
+DBT can read a configuration file located at:
+- `$DBT_CONFIG_DIR/config.yaml`, if `$DBT_CONFIG_DIR` is set.
+- `$XDG_CONFIG_DIR/dbt/config.yaml`, if `$XDG_CONFIG_DIR` is set.
+- `$HOME/.config/dbt/config.yaml`, if `$HOME` is set.
+- Otherwise no configuration is loaded.
+
+This configuration file allows the user to configure a DBT local mirror with the following line:
+
+```yaml
+mirror: "<PATH_TO_YOUR_LOCAL_MIRROR>"
+```
+
+Replace `<PATH_TO_YOUR_LOCAL_MIRROR>` by a full path in your system where your user has read and
+write permissions. Note that this path MUST exist for the mirror to be used by DBT.
+
+With a local mirror configured, DBT will reduce the amount of bandwidth required to sync dependencies.
+In particular, its behavior is different between archives and git repositories:
+- Compressed archives (`*.tar.gz`): they get downloaded first into the local mirror and then
+copied to your project's dependency folder. If they are already available in your local mirror, they
+are simply copied over to your dependency folder, so no network access is required.
+- Git repositories: they get cloned with the `--mirror` flag in the mirror directory. In your dependency
+folder, they get cloned using the url and a `--reference` flag pointing to the local mirror. For them,
+some network access might be required (e.g., if the branch has updates), but the bulk of fetching
+all git objects can be done from the local mirror.
```

Comment:
I would add something about having to manually clean up the cache every now and then. For the bazel cache I'm using `find $cachedir -atime +7 | xargs rm` which deletes everything that hasn't been accessed in the last 7 days. Maybe something similar can be useful here too.

Reply:
That seems slightly dangerous. If a file has not been accessed in the a while it does not mean it is not needed. I think this could be rather problematic with git objects. But I will add a warning nonetheless about cleaning up the cache (manually)

Reply:
Well you never truly know if it might be needed again at some point in the future when you are checking out a commit from 6 months ago. But indeed, for git objects it's not a good idea. In this case it would need to apply to the whole repository or tarball...